src/MimirCommander.cpp

```diff
@@ -0,0 +1,48 @@
+//
+// Created by rolf on 27-10-19.
+//
+
+#include "MimirCommander.h"
+#include "iostream"
+namespace rtt{
+namespace robothub{
+void MimirCommander::setColor(bool isYellow) {
```

Comment:
make sure you format your code correctly. spaces/newlines are different everywhere